src/pages/instances/InstanceConsole.tsx

```diff
@@ -29,6 +45,40 @@ const InstanceConsole: FC<Props> = ({ instance }) => {
     handleFullScreen = childHandleFullScreen;
   };
 
+  const switchConsole = (isGraphic: boolean) => {
+    setInTabNotification(null);
+    setGraphic(isGraphic);
+  };
+
+  const handleStart = () => {
```

Comment:
I wonder if we can share this code with the StartInstanceButton. Especially with all the isLoading/isRunning calculations and the useNotify, useQueryClient and useInstanceLoading bits - would be good to have them in one place only.

Reply:
I already thought about this, and could not find a straightforward solution. Any proposal?

Reply:
Seems right in combination with the empty state, but getting a non-interactive text console with the logs -- not sure if you'd need an error or not to clarify why it is not interactive.

Reply:
You can directly use the `StartInstanceButton` in here and give it a single prop `btnProps: ButtonProps` that is spread into the Button. It will allow styling it to match this use cases layout.